addons/sms/tools/sms_tools.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import re
+
+from odoo.tools import html_keep_url, misc
```

Comment:
Note that you can import html_escape directly from odoo.tools (odoo.tools import all sub-tools)